index.html

```diff
@@ -1,246 +1,139 @@
-            opening
-          </p>
-        </div>
-      </div>
-      <button class="join-Button" type="button">
-        Join CC Global Summit 2015
-      </button>
-
-      <div class="prg1">
-        <a class="program1">SEE THE WHOLE PROGRAM</a>
-      </div>
-    </section>
-
-    <section class="page2">
-      <div class="prg1">
-        <h3 class="program2">Featured Speakers</h3>
-        <hr class="underline" />
-      </div>
-
-      <div class="speakerCards">
-        <div class="card-lec">
-          <img src="./images/Picture3.png" alt="lecture" />
-
-          <div class="feature-card">
-            <h3>FirstName LastName</h3>
-            <h5>
-              Bickerman Professor of Entreprenurial Legal Studies at Havard Law
-              School
-            </h5>
-            <p>
-              Listen to the speaker from various countries about sharing and
-              opening
+    <section class="banner">
+        <div class="bcontent">
+            <p class="bgreeting">"Microverse Shares Knowledge!"</p>
+            <h1>
+                MICROVERSE <br>
+                VERSION 2 PROGRAM<br>
+                2022
+            </h1>
+            <p class="bdesc">Microverse plans to make most people from all over the word with remote jobs by 2030. The Microverse event will be an opportunity to learn about what's in the program and become a successful Software Developer.
             </p>
-          </div>
+            <h2>2022-05-20(FRI) ~ 28(SAT)</h2>
+            <h4>@ Microverse, California, USA</h4>
         </div>
-        <div class="card-lec">
-          <img src="./images/Picture3.png" alt="lecture" />
-
-          <div class="feature-card">
-            <h3>Lecturer</h3>
-            <h5>Speaker</h5>
-            <p>
-              Listen to the speaker from various countries about sharing and
-              opening
-            </p>
-          </div>
-        </div>
-        <div class="dis-none">
-          <button class="show-more" type="button">MORE &#x2228;</button>
-        </div>
-      </div>
-
-      <div class="speakerCards1 dis-none1">
-        <div class="card-lec">
-          <img src="./images/Picture3.png" alt="lecture" />
-          <div class="feature-card">
-            <h3>Lecturer1</h3>
-            <h5>Speaker</h5>
-            <p>
-              Listen to the speaker from various countries about sharing and
-              opening
-            </p>
-          </div>
-        </div>
-
-        <div class="card-lec">
-          <img src="./images/Picture3.png" alt="lecture" />
-
-          <div class="feature-card">
-            <h3>Lecturer2</h3>
-            <h5>Speaker</h5>
-            <p>
-              Listen to the speaker from various countries about sharing and
-              opening
-            </p>
-          </div>
+    </section>
+    <section class="mainpro" id="pro">
+        <h2 class="proheader">Main Program</h2>
```

Comment:
- [ ] [OPTIONAL] In the design we can see that after this header there is a line that's an indicator, kindly add it to your page.

### Behance Design
![image](https://user-images.githubusercontent.com/59416136/169030739-c08dd54e-5c2d-4458-abb6-8ce05dd70a91.png)

![image](https://user-images.githubusercontent.com/59416136/169030808-e878c186-b862-437b-906c-2668ffd707d8.png)

![image](https://user-images.githubusercontent.com/59416136/169030884-1b172914-a250-45fc-8b7b-0321deacdb5d.png)

### Your Implementation
![image](https://user-images.githubusercontent.com/59416136/169030973-900fb6f9-90d3-43a9-a8f5-1db607b22138.png)

![image](https://user-images.githubusercontent.com/59416136/169031020-cf5281ad-2c92-4df2-acd5-1d0c5eac744e.png)

Reply:
ok thanks for this just perfect and helpful advice, just update all changes needed for my third review request